charts/sysdig-deploy/README.md

```diff
@@ -168,7 +168,7 @@ helm upgrade -n sysdig-agent sysdig sysdig/sysdig-deploy -f values.sysdig.yaml
 
 ## Configuration
 
-The following table lists the configurable parameters of this chart and their default values.
+The following table lists the configurable parameters of this chart and their default values, it is indicative and contains only the most commonly used parameters. If you are looking for more advanced configuration values, those are available in the documentation of the admission-controller, agent, node-analyzer, kspm-collector and rapid-response charts.
```

Comment:
If I recall correctly, the only parameters/values we should be putting in the `sysdig-deploy` chart are those that can be used by multiple charts, or a simple boolean to enable/disable a given subchart. As such, I would recommend rephrasing this whole block similar to the following:

The following table lists the configurable parameters of the sysdig-deploy chart and their default values.  The sysdig-deploy chart itself only has select parameters that are used by multiple subcharts, and those used to enable/disable select subcharts. If you need additional configuration values, those are available in the various READMEs of the individual subcharts.

If you wanted to get fancy you could add links to the existing READMEs too.

Reply:
Thanks, I included your suggestion and added links as well :)